### ISSUE TITLE
get_bank_loading_state() doesn't correctly return bank loading state if bank is unloaded

### ISSUE THREAD

**Issue by 2shady4u** [utopia-rise/fmod-gdnative#76]
Relevant part is in `test_bank.gd`:
![image](https://user-images.githubusercontent.com/42484461/107563180-0889e300-6be1-11eb-99f2-7c04c81c4ede.png)

get_bank_loading_state() should return -1, but it crashes instead 🤷‍♂️ 

![image](https://user-images.githubusercontent.com/42484461/107563294-2e16ec80-6be1-11eb-9b9f-8509a7260b6e.png)

**Comment by 2shady4u** [comment]
Leakage with verbose output:

```
ERROR: ~List: Condition "_first != __null" is true.
   At: ./core/self_list.h:112
ERROR: ~List: Condition "_first != __null" is true.
   At: ./core/self_list.h:112
WARNING: cleanup: ObjectDB instances leaked at exit (run with --verbose for details).
     At: core/object.cpp:2135
Leaked instance: _File:1520
Leaked instance: Node:1775 - Node name: @@30
Leaked instance: _File:1666
Leaked instance: Reference:1841
Leaked instance: Node2D:1383 - Node name:
Leaked instance: Reference:1779
Leaked instance: Reference:1672
Leaked instance: Node:1670 - Node name: @@25
Leaked instance: Sprite:1364 - Node name:
Leaked instance: Node:1357 - Node name:
Leaked instance: Reference:1788
Leaked instance: _File:1787
Leaked instance: Reference:1867
Leaked instance: Reference:1448
Leaked instance: Node:1535 - Node name: @@16
Leaked instance: Reference:1420
Leaked instance: Node2D:1676 - Node name:
Leaked instance: Node:1784 - Node name: @@31
Leaked instance: Reference:1776
Leaked instance: Sprite:1451 - Node name:
Leaked instance: Sprite:1346 - Node name:
Leaked instance: _File:1818
Leaked instance: Reference:1564
Leaked instance: GDScript:1829 - Resource path: res://test/unit/test_sound.gd
Leaked instance: Node2D:1399 - Node name:
Leaked instance: Reference:1563
Leaked instance: _File:1522
Leaked instance: GDScript:1464 - Resource path:
Leaked instance: _File:1735
Leaked instance: Reference:1855
Leaked instance: GDScriptNativeClass:542
Leaked instance: _File:1664
Leaked instance: Reference:1566
Leaked instance: _File:1362
Leaked instance: Reference:1806
Leaked instance: Node:1661 - Node name: @@24
Leaked instance: Reference:1723
Leaked instance: Node2D:1667 - Node name:
Leaked instance: Node2D:1240 - Node name:
Leaked instance: Node2D:1488 - Node name:
Leaked instance: _File:1396
Leaked instance: Reference:1467
Leaked instance: Reference:1446
Leaked instance: Node2D:1265 - Node name:
Leaked instance: _File:1449
Leaked instance: _File:1673
Leaked instance: _File:1501
Leaked instance: Node2D:1559 - Node name:
Leaked instance: Reference:1530
Leaked instance: _File:1538
Leaked instance: Node2D:1259 - Node name:
Leaked instance: _File:1809
Leaked instance: Reference:1865
Leaked instance: _File:1422
Leaked instance: Reference:1343
Leaked instance: Node2D:1504 - Node name:
Leaked instance: _File:1380
Leaked instance: GDScript:1525 - Resource path: res://test/unit/test_bank.gd
Leaked instance: Reference:1808
Leaked instance: GDScript:1879 - Resource path: res://test/unit/test_zclose_fmod.gd
Leaked instance: Reference:1864
Leaked instance: GDScript:1717 - Resource path: res://test/unit/test_event.gd
Leaked instance: Reference:1341
Leaked instance: Node2D:1264 - Node name:
Leaked instance: Reference:1554
Leaked instance: GDScript:1799 - Resource path: res://test/unit/test_listener.gd
Leaked instance: Node:1517 - Node name: @@14
Leaked instance: Node2D:1425 - Node name:
Leaked instance: Node:1498 - Node name:
Leaked instance: Reference:1555
Leaked instance: _File:1724
Leaked instance: _File:1470
Leaked instance: Reference:1734
Leaked instance: Reference:1359
Leaked instance: Reference:1527
Leaked instance: Reference:1722
Leaked instance: Reference:1732
Leaked instance: Reference:1469
Leaked instance: Reference:1445
Leaked instance: Node2D:1869 - Node name:
Leaked instance: Node2D:1736 - Node name:
Leaked instance: Reference:1499
Leaked instance: Node2D:1471 - Node name:
Leaked instance: _File:1726
Leaked instance: GDScript:1337 - Resource path:
Leaked instance: Reference:1882
Leaked instance: Node:1553 - Node name: @@18
Leaked instance: GDScript:1497 - Resource path:
Leaked instance: _File:1468
Leaked instance: Sprite:1472 - Node name:
Leaked instance: _File:1447
Leaked instance: Reference:1557
Leaked instance: Reference:1528
Leaked instance: GDScriptNativeClass:687
Leaked instance: Node:1338 - Node name:
Leaked instance: Reference:1881
Leaked instance: GDScript:1356 - Resource path:
Leaked instance: GDScript:1853 - Resource path: res://test/unit/test_vca.gd
Leaked instance: Node:1526 - Node name: @@15
Leaked instance: Reference:1671
Leaked instance: _Mutex:1184
Leaked instance: Node2D:1523 - Node name:
Leaked instance: Node:1393 - Node name:
Leaked instance: Reference:1663
Leaked instance: Reference:1502
Leaked instance: _File:1382
Leaked instance: GDScript:1443 - Resource path:
Leaked instance: Reference:1466
Leaked instance: _File:1857
Leaked instance: Node:1721 - Node name: @@27
Leaked instance: Node:1444 - Node name:
Leaked instance: Reference:1725
Leaked instance: _File:1531
Leaked instance: _File:1503
Leaked instance: _File:1556
Leaked instance: GDScript:1195 - Resource path: res://addons/gut/doubler.gd
Leaked instance: Reference:1423
Leaked instance: GDScript:1192 - Resource path: res://addons/gut/utils.gd
Leaked instance: Node2D:1886 - Node name:
Leaked instance: Node:1419 - Node name:
Leaked instance: _File:1675
Leaked instance: GDScript:1194 - Resource path: res://addons/gut/logger.gd
Leaked instance: GDScript:1198 - Resource path: res://addons/gut/method_maker.gd
Leaked instance: Reference:1858
Leaked instance: GDScript:1201 - Resource path: res://addons/gut/stub_params.gd
Leaked instance: Reference:1731
Leaked instance: Node:1465 - Node name:
Leaked instance: _File:1885
Leaked instance: GDScript:1552 - Resource path: res://test/unit/test_bus.gd
Leaked instance: GDScript:1197 - Resource path:
Leaked instance: GDScriptNativeClass:675
Leaked instance: Reference:1674
Leaked instance: GDScript:1205 - Resource path: res://addons/gut/test.gd
Leaked instance: _File:1842
Leaked instance: Reference:1358
Leaked instance: GDScript:1392 - Resource path:
Leaked instance: Node:1854 - Node name: @@38
Leaked instance: Node2D:1845 - Node name:
Leaked instance: GDScript:1202 - Resource path: res://addons/gut/summary.gd
Leaked instance: Sprite:1505 - Node name:
Leaked instance: GDScript:1199 - Resource path: res://addons/gut/spy.gd
Leaked instance: Node2D:1450 - Node name:
Leaked instance: Node2D:1532 - Node name:
Leaked instance: Reference:1394
Leaked instance: _File:1883
Leaked instance: Reference:1395
Leaked instance: _File:1342
Leaked instance: Reference:1834
Leaked instance: GDNativeLibrary:1177 - Resource path: res://addons/fmod/Fmod.gdnlib
Leaked instance: Node2D:1363 - Node name:
Leaked instance: Node:1804 - Node name: @@33
Leaked instance: Node2D:1727 - Node name:
Leaked instance: Reference:1397
Leaked instance: Node:1863 - Node name: @@39
Leaked instance: Reference:1518
Leaked instance: Reference:1665
Leaked instance: Node2D:1810 - Node name:
Leaked instance: Reference:1537
Leaked instance: Reference:1421
Leaked instance: _File:1866
Leaked instance: _File:1360
Leaked instance: _File:1558
Leaked instance: GDScript:1516 - Resource path: res://test/unit/test_ainit_fmod.gd
Leaked instance: Reference:1379
Leaked instance: Node2D:1245 - Node name:
Leaked instance: Reference:1361
Leaked instance: _File:1733
Leaked instance: Node:1880 - Node name: @@41
Leaked instance: GDScript:1418 - Resource path:
Leaked instance: Node:1562 - Node name: @@19
Leaked instance: Node:1813 - Node name: @@34
Leaked instance: Node2D:1568 - Node name:
Leaked instance: Reference:1521
Leaked instance: Reference:1662
Leaked instance: Node2D:1252 - Node name:
Leaked instance: _File:1780
Leaked instance: _File:1529
Leaked instance: Reference:1500
Leaked instance: Sprite:1426 - Node name:
Leaked instance: Sprite:1384 - Node name:
Leaked instance: GDScript:1339 - Resource path: res://addons/gut/signal_watcher.gd
Leaked instance: Node2D:1260 - Node name:
Leaked instance: Node:1730 - Node name: @@28
Leaked instance: Node2D:1836 - Node name:
Leaked instance: Node2D:1345 - Node name:
Leaked instance: _File:1565
Leaked instance: GDScript:1376 - Resource path:
Leaked instance: _File:1789
Leaked instance: Node:1613 - Node name: @@22
Leaked instance: Node2D:1860 - Node name:
Leaked instance: _File:1835
Leaked instance: NativeScript:1179 - Resource path: res://addons/fmod/Fmod.gdns
Leaked instance: Reference:1615
Leaked instance: _File:1609
Leaked instance: GDScript:1207 - Resource path: res://addons/gut/test_collector.gd
Leaked instance: _File:1778
Leaked instance: _File:1607
Leaked instance: _File:1540
Leaked instance: Reference:1785
Leaked instance: GDScript:1774 - Resource path: res://test/unit/test_global.gd
Leaked instance: Node2D:1541 - Node name:
Leaked instance: Reference:1484
Leaked instance: Reference:1856
Leaked instance: Node:1839 - Node name: @@37
Leaked instance: _File:1618
Leaked instance: GDScript:1210 - Resource path: res://addons/gut/thing_counter.gd
Leaked instance: Reference:1608
Leaked instance: GDScript:1603 - Resource path: res://test/unit/test_callbacks.gd
Leaked instance: _File:1344
Leaked instance: Node2D:1790 - Node name:
Leaked instance: _File:1844
Leaked instance: Reference:1777
Leaked instance: _File:1616
Leaked instance: Reference:1605
Leaked instance: Reference:1786
Leaked instance: Node2D:1610 - Node name:
Leaked instance: Reference:1606
Leaked instance: Node2D:1619 - Node name:
Leaked instance: Node2D:1781 - Node name:
Leaked instance: Reference:1843
Leaked instance: Reference:1617
Leaked instance: Reference:1483
Leaked instance: _File:1398
Leaked instance: Reference:1840
Leaked instance: _File:1868
Leaked instance: Reference:1815
Leaked instance: Reference:1614
Leaked instance: _File:1487
Leaked instance: Sprite:1400 - Node name:
Leaked instance: Reference:1832
Leaked instance: Reference:1831
Leaked instance: _File:1485
Leaked instance: Reference:1884
Leaked instance: Node:1183 - Node name:
Leaked instance: Node2D:1269 - Node name:
Leaked instance: GDScript:1481 - Resource path:
Leaked instance: ConfigFile:1178
Leaked instance: _File:1424
Leaked instance: Reference:1817
Leaked instance: GDScript:1209 - Resource path:
Leaked instance: Reference:1340
Leaked instance: Reference:1539
Leaked instance: Reference:1536
Leaked instance: Reference:1814
Leaked instance: GDScript:1206 - Resource path:
Leaked instance: _File:1567
Leaked instance: GDScript:1203 - Resource path:
Leaked instance: Node:1482 - Node name:
Leaked instance: GDScript:1200 - Resource path: res://addons/gut/stubber.gd
Leaked instance: Reference:1381
Leaked instance: Node:1377 - Node name:
Leaked instance: GDScript:1196 - Resource path:
Leaked instance: _File:1816
Leaked instance: Node2D:1819 - Node name:
Leaked instance: Node:1830 - Node name: @@36
Leaked instance: _File:1807
Leaked instance: GDScript:1660 - Resource path: res://test/unit/test_desc_event.gd
Leaked instance: Node:1604 - Node name: @@21
Leaked instance: _File:1859
Leaked instance: _File:1833
Leaked instance: Reference:1805
Leaked instance: Reference:1486
Leaked instance: Reference:1519
Leaked instance: Reference:1378
Leaked instance: GDScript:1204 - Resource path:
Leaked instance: GDScript:1208 - Resource path:
Hint: Leaked instances typically happen when nodes are removed from the scene tree (with `remove_child()`) but not freed (with `free()` or `queue_free()`).
ERROR: clear: Resources still in use at exit (run with --verbose for details).
   At: core/resource.cpp:477
Resource still in use: res://addons/gut/thing_counter.gd (GDScript)
Resource still in use: res://addons/gut/stub_params.gd (GDScript)
Resource still in use: res://addons/gut/utils.gd (GDScript)
Resource still in use: res://addons/fmod/Fmod.gdnlib (GDNativeLibrary)
Resource still in use: res://test/unit/test_bank.gd (GDScript)
Resource still in use: res://test/unit/test_zclose_fmod.gd (GDScript)
Resource still in use: res://addons/gut/signal_watcher.gd (GDScript)
Resource still in use: res://test/unit/test_listener.gd (GDScript)
Resource still in use: res://addons/gut/spy.gd (GDScript)
Resource still in use: res://addons/gut/doubler.gd (GDScript)
Resource still in use: res://addons/fmod/Fmod.gdns (NativeScript)
Resource still in use: res://test/unit/test_desc_event.gd (GDScript)
Resource still in use: res://test/unit/test_callbacks.gd (GDScript)
Resource still in use: res://addons/gut/test_collector.gd (GDScript)
Resource still in use: res://addons/gut/summary.gd (GDScript)
Resource still in use: res://test/unit/test_sound.gd (GDScript)
Resource still in use: res://test/unit/test_global.gd (GDScript)
Resource still in use: res://addons/gut/stubber.gd (GDScript)
Resource still in use: res://test/unit/test_vca.gd (GDScript)
Resource still in use: res://test/unit/test_event.gd (GDScript)
Resource still in use: res://test/unit/test_bus.gd (GDScript)
Resource still in use: res://test/unit/test_ainit_fmod.gd (GDScript)
Resource still in use: res://addons/gut/test.gd (GDScript)
Resource still in use: res://addons/gut/method_maker.gd (GDScript)
Resource still in use: res://addons/gut/logger.gd (GDScript)
Orphan StringName: res://addons/gut/doubler.gd::360::set_logger
Orphan StringName: Node2D
Orphan StringName: _pending
Orphan StringName: get_sections
Orphan StringName: reason
Orphan StringName: ::57::TestEventDescription.test_assert_should_not_be_snapshot
Orphan StringName: set_yield_time
Orphan StringName: res://addons/gut/test.gd::618::get_call_parameters
Orphan StringName: copy
Orphan StringName: ::51::TestEventDescription.test_assert_should_be_3d
Orphan StringName: res://addons/gut/utils.gd::0::_init
Orphan StringName: DoubleInfo
Orphan StringName: assert_extends
Orphan StringName: get_deprecated
Orphan StringName: _file_checker
Orphan StringName: expected_count
Orphan StringName: txt
Orphan StringName: set_use_unique_names
Orphan StringName: res://addons/gut/thing_counter.gd::12::has
Orphan StringName: check_VCA_path
Orphan StringName: get_pending_count
Orphan StringName: ::8::TestVCA.before_all
Orphan StringName: ::69::TestEvent.test_assert_event_parameter_by_name
Orphan StringName: to_s
Orphan StringName: res://addons/gut/spy.gd::41::was_called
Orphan StringName: set_VCA_volume
Orphan StringName: sound
Orphan StringName: res://addons/gut/utils.gd::58::join_array
Orphan StringName: res://addons/gut/test_collector.gd::192::get_test_prefix
Orphan StringName: res://addons/gut/doubler.gd::384::partial_double_inner
Orphan StringName: ::34::TestVCA.test_assert_invalid_path
Orphan StringName: get_sound_volume
Orphan StringName: get_bus_volume
Orphan StringName: free
Orphan StringName: ::102::ObjectInfo.get_extends_text
Orphan StringName: res://addons/gut/test.gd::166::_do_datatypes_match__fail_if_not
Orphan StringName: ::11::TestSound.before_all
Orphan StringName: desc_is_3d
Orphan StringName: parameters
Orphan StringName: _lgr
Orphan StringName: get_node
Orphan StringName: TestCollector
Orphan StringName: res://addons/gut/stubber.gd::76::_find_stub
Orphan StringName: remove_child
Orphan StringName: get_signal_emit_count
Orphan StringName: watch_signal
Orphan StringName: NativeScript
Orphan StringName: set_name
Orphan StringName: add_pending
Orphan StringName: __gut_metadata_
Orphan StringName: res://addons/gut/logger.gd::0::_init
Orphan StringName: scene_info
Orphan StringName: get_debugs
Orphan StringName: get_method_list
Orphan StringName: ::39::TestScript.get_pending_count
Orphan StringName: extend
Orphan StringName: res://addons/gut/doubler.gd::287::_get_temp_path
Orphan StringName: set_listener_weight
Orphan StringName: call_method
Orphan StringName: get_event_paused
Orphan StringName: _fail_if_does_not_have_signal
Orphan StringName: res://addons/gut/stub_params.gd::20::to_do_nothing
Orphan StringName: _find_object_property
Orphan StringName: get_spy
Orphan StringName: get_value
Orphan StringName: ::99::ObjectInfo.has_subpath
Orphan StringName: ::22::TestCallbacks.after_all
Orphan StringName: desc_is_stream
Orphan StringName: res://addons/gut/summary.gd::83::add_fail
Orphan StringName: res://addons/gut/test_collector.gd
Orphan StringName: to_split
Orphan StringName: ::54::TestEventDescription.test_assert_should_not_be_oneshot
Orphan StringName: _fail
Orphan StringName: res://addons/gut/logger.gd::23::_gut_log_level_for_type
Orphan StringName: ::89::TestEvent.test_assert_attached_to_node
Orphan StringName: TestCallbacks
Orphan StringName: method
Orphan StringName: dict
Orphan StringName: add_child
Orphan StringName: res://addons/gut/test_collector.gd::204::clear
Orphan StringName: ::73::TestEvent.test_assert_should_pause_all
Orphan StringName: end_test
Orphan StringName: set_event_timeline_position
Orphan StringName: ::36::ScriptMethods._add_name_if_does_not_have
Orphan StringName: all_params
Orphan StringName: inner_name
Orphan StringName: ::0::TestBus._init
Orphan StringName: _subpaths
Orphan StringName: res://addons/gut/logger.gd::48::get_errors
Orphan StringName: subpath
Orphan StringName: _on_watched_signal
Orphan StringName: major
Orphan StringName: res://addons/gut/test.gd::140::_fail
Orphan StringName: res://addons/gut/logger.gd::45::get_warnings
Orphan StringName: get_event_parameter_by_name
Orphan StringName: test_assert_playing
Orphan StringName: test_assert_should_not_be_snapshot
Orphan StringName: fail_count
Orphan StringName: ::79::TestScript.import_from
Orphan StringName: res://addons/gut/test.gd::271::assert_eq
Orphan StringName: Summary
Orphan StringName: is_not_freed
Orphan StringName: get_test_obj
Orphan StringName: method_name
Orphan StringName: _log
Orphan StringName: test_assert_event_parameter_by_name
Orphan StringName: inner_class_name
Orphan StringName: res://addons/gut/doubler.gd::378::double
Orphan StringName: to_replace
Orphan StringName: desc_get_minimum_distance
Orphan StringName: msg
Orphan StringName: meta
Orphan StringName: res://addons/gut/doubler.gd::340::set_output_dir
Orphan StringName: from
Orphan StringName: instance
Orphan StringName: get_as_text
Orphan StringName: _make_key_from_metadata
Orphan StringName: res://addons/gut/method_maker.gd::67::_init
Orphan StringName: found
Orphan StringName: desired_listener
Orphan StringName: add_stub
Orphan StringName: ::55::TestListener.test_assert_attach_object_to_listener
Orphan StringName: error
Orphan StringName: clear_output_directory
Orphan StringName: res://addons/gut/summary.gd::89::get_test_text
Orphan StringName: is_watching
Orphan StringName: res://addons/gut/test.gd::238::before_all
Orphan StringName: res://addons/gut/test.gd::220::_get_fail_msg_including_emitted_signals
Orphan StringName: max_wait
Orphan StringName: scene_path
Orphan StringName: FMOD_INIT_NORMAL
Orphan StringName: ObjectInfo
Orphan StringName: plus_file
Orphan StringName: sound_played
Orphan StringName: get_name_count
Orphan StringName: ::26::TestEvent.after_all
Orphan StringName: res://addons/gut/doubler.gd::388::double_inner
Orphan StringName: res://addons/gut/test.gd::449::assert_accessors
Orphan StringName: is_watching_object
Orphan StringName: _signal_watcher
Orphan StringName: before_all
Orphan StringName: res://test/unit/test_zclose_fmod.gd::3::before_all
Orphan StringName: _use_unique_names
Orphan StringName: assert_file_empty
Orphan StringName: ::55::TestScript.has_inner_class
Orphan StringName: script_path
Orphan StringName: get_test_text
Orphan StringName: get_bank_event_count
Orphan StringName: get_count
Orphan StringName: inner_classes
Orphan StringName: if_null
Orphan StringName: get_assert_count
Orphan StringName: ::74::TestEventDescription.test_assert_sound_size
Orphan StringName: ::9::TestBus.before_all
Orphan StringName: params
Orphan StringName: ::59::TestScript.add_pending
Orphan StringName: expect_high
Orphan StringName: set_event_reverb_level
Orphan StringName: test_prefix
Orphan StringName: get_call_parameters
Orphan StringName: res://addons/gut/thing_counter.gd::41::add_array_items
Orphan StringName: res://addons/gut/spy.gd::92::get_logger
Orphan StringName: ::33::TestScript.get_fail_count
Orphan StringName: res://addons/gut/spy.gd::81::clear
Orphan StringName: has_unsupported_defaults
Orphan StringName: ::29::TestListener.test_assert_should_set_listener_num
Orphan StringName: ::45::TestScript.get_test_obj
Orphan StringName: res://addons/gut/test.gd::536::assert_signal_not_emitted
Orphan StringName: res://addons/gut/test_collector.gd::198::get_test_class_prefix
Orphan StringName: _fail_pass_text
Orphan StringName: arg1
Orphan StringName: ::78::ObjectInfo.instantiate
Orphan StringName: arg2
Orphan StringName: arg3
Orphan StringName: arg4
Orphan StringName: arg5
Orphan StringName: arg6
Orphan StringName: res://addons/gut/test.gd::809::pending
Orphan StringName: arg7
Orphan StringName: arg8
Orphan StringName: ::52::TestScript.get_filename
Orphan StringName: arg9
Orphan StringName: res://addons/gut/stubber.gd::123::get_return
Orphan StringName: stub_method
Orphan StringName: get_instance_id
Orphan StringName: assert_signal_emit_count
Orphan StringName: res://addons/gut/doubler.gd::129::_init
Orphan StringName: add_array_items
Orphan StringName: TestScript
Orphan StringName: section
Orphan StringName: res://addons/gut/stubber.gd::27::_make_key_from_variant
Orphan StringName: call_size
Orphan StringName: search
Orphan StringName: Spy
Orphan StringName: get_event_reverb_level
Orphan StringName: FMOD_STUDIO_PLAYBACK_STOPPED
Orphan StringName: res://addons/gut/signal_watcher.gd::98::does_object_have_signal
Orphan StringName: res://addons/gut/test.gd::919::double
Orphan StringName: res://test/unit/test_listener.gd::0::_init
Orphan StringName: get_decleration_text
Orphan StringName: PARTIAL
Orphan StringName: args
Orphan StringName: Reference
Orphan StringName: TestGlobal
Orphan StringName: deprecated
Orphan StringName: ::32::TestBank.test_assert_bank_bus_count
Orphan StringName: TestBank
Orphan StringName: remap_path
Orphan StringName: _is_supported_default
Orphan StringName: stub_info
Orphan StringName: NATIVE_CLASS
Orphan StringName: res://addons/gut/test_collector.gd::181::to_s
Orphan StringName: _smart_double
Orphan StringName: res://addons/gut/signal_watcher.gd::57::_add_watched_signal
Orphan StringName: res://addons/gut/utils.gd::26::is_version_31
Orphan StringName: res://addons/gut/test.gd::855::get_pass_count
Orphan StringName: assert_does_not_have
Orphan StringName: _add_name_if_does_not_have
Orphan StringName: res://addons/gut/spy.gd
Orphan StringName: local_methods
Orphan StringName: ::0::TestEventDescription._init
Orphan StringName: res://test/unit/test_desc_event.gd::0::_init
Orphan StringName: ::29::TestGlobal.test_assert_should_have_cpu_performance_data
Orphan StringName: assert_listener_num
Orphan StringName: invalid
Orphan StringName: test_assert_set_callback
Orphan StringName: ::35::TestEvent.test_should_has_event
Orphan StringName: error_interval
Orphan StringName: join_array
Orphan StringName: disp
Orphan StringName: FMOD_STUDIO_PLAYBACK_PLAYING
Orphan StringName: shutdown
Orphan StringName: LoadedClass
Orphan StringName: ::24::TestBank.after_all
Orphan StringName: _add_obj_method
Orphan StringName: empty_search
Orphan StringName: get_event_playback_state
Orphan StringName: cpu_perf_data
Orphan StringName: _get_desc_of_calls_to_instance
Orphan StringName: should_add
Orphan StringName: add_built_in_method
Orphan StringName: _scripts
Orphan StringName: parent
Orphan StringName: _output_dir
Orphan StringName: res://addons/gut/logger.gd::69::get_log_entries
Orphan StringName: split_string
Orphan StringName: get_root
Orphan StringName: test_assert_should_create_and_release
Orphan StringName: res://addons/gut/doubler.gd::209::_get_methods
Orphan StringName: #match_value
Orphan StringName: get_unique_count
Orphan StringName: did_emit
Orphan StringName: res://addons/gut/test.gd::596::get_signal_emit_count
Orphan StringName: res://addons/gut/signal_watcher.gd::159::get_signals_emitted
Orphan StringName: clear_signal_watcher
Orphan StringName: ::0::TestVCA._init
Orphan StringName: fail_texts
Orphan StringName: get_filename
Orphan StringName: stub_params
Orphan StringName: res://addons/gut/test.gd::250::after_each
Orphan StringName: pending
Orphan StringName: override_strat
Orphan StringName: call
Orphan StringName: res://addons/gut/test.gd::864::clear_signal_watcher
Orphan StringName: desc_get_maximum_distance
Orphan StringName: set_bus_volume
Orphan StringName: _double_scene_and_script
Orphan StringName: res://addons/gut/thing_counter.gd
Orphan StringName: res://addons/gut/doubler.gd::348::set_spy
Orphan StringName: get_pass_count
Orphan StringName: test_name
Orphan StringName: is_file_empty
Orphan StringName: _gut_log_level_for_type
Orphan StringName: res://addons/gut/method_maker.gd::196::get_logger
Orphan StringName: assert_not_null
Orphan StringName: res://addons/gut/test.gd::724::assert_called
Orphan StringName: new
Orphan StringName: ::37::TestSound.test_assert_playing
Orphan StringName: res://addons/gut/test_collector.gd::228::import_tests
Orphan StringName: ::55::TestScript.add_fail
Orphan StringName: ::9::TestEvent.before_all
Orphan StringName: _get_arg_text
Orphan StringName: ::9::TestEventDescription.before_all
Orphan StringName: test_assert_should_have_proper_weight
Orphan StringName: res://addons/gut/doubler.gd::364::get_strategy
Orphan StringName: ::54::TestBank.test_assert_bank_vca_count
Orphan StringName: remove
Orphan StringName: res://addons/gut/test.gd::630::assert_extends
Orphan StringName: _utils
Orphan StringName: res://addons/gut/test.gd::434::assert_has_method
Orphan StringName: res://addons/gut/stub_params.gd::23::to_call_super
Orphan StringName: GDScriptNativeClass
Orphan StringName: res://addons/gut/test.gd::413::assert_file_empty
Orphan StringName: default
Orphan StringName: calls
Orphan StringName: was_called
Orphan StringName: totals
Orphan StringName: ::0::TestListener._init
Orphan StringName: iclass_name
Orphan StringName: res://test/unit/test_callbacks.gd::0::_init
Orphan StringName: export_to
Orphan StringName: res://addons/gut/test_collector.gd::165::add_script
Orphan StringName: res://addons/gut/utils.gd::98::file_exists
Orphan StringName: _can_make_signal_assertions
Orphan StringName: assert_called
Orphan StringName: READ
Orphan StringName: FMOD_STUDIO_EVENT_CALLBACK_ALL
Orphan StringName: info
Orphan StringName: test_assert_paused
Orphan StringName: res://addons/gut/stubber.gd::68::add_stub
Orphan StringName: config_file
Orphan StringName: res://addons/gut/test_collector.gd::107::_parse_script
Orphan StringName: _get_inst_id_ref_str
Orphan StringName: FMOD_SPEAKERMODE_STEREO
Orphan StringName: res://addons/gut/logger.gd::88::deprecated
Orphan StringName: ::54::TestBus.test_assert_volume
Orphan StringName: a_class
Orphan StringName: get_event_pitch
Orphan StringName: set_sound_3D_settings
Orphan StringName: res://addons/gut/test_collector.gd::217::export_tests
Orphan StringName: ::60::TestEventDescription.test_assert_should_not_be_stream
Orphan StringName: line_len
Orphan StringName: get_scripts
Orphan StringName: get_output_dir
Orphan StringName: _double_script
Orphan StringName: ::32::TestBus.test_assert_should_has_master_bus
Orphan StringName: test_assert_set_volume
Orphan StringName: res://addons/gut/test.gd::187::_get_desc_of_calls_to_instance
Orphan StringName: get_file
Orphan StringName: ::71::TestListener.assert_listener_weight
Orphan StringName: debug
Orphan StringName: _get_params_as_string
Orphan StringName: test_assert_sound_size
Orphan StringName: init
Orphan StringName: res://addons/gut/test.gd::402::assert_file_does_not_exist
Orphan StringName: get_signals_emitted
Orphan StringName: null_idx
Orphan StringName: mem_perf_data
Orphan StringName: res://addons/gut/signal_watcher.gd::117::get_emit_count
Orphan StringName: res://addons/gut/summary.gd::86::add_pending
Orphan StringName: res://addons/gut/stub_params.gd::10::_init
Orphan StringName: ::84::ObjectInfo.get_loaded_class
Orphan StringName: _do_datatypes_match__fail_if_not
Orphan StringName: assert_string_contains
Orphan StringName: tests
Orphan StringName: names
Orphan StringName: get_script
Orphan StringName: set_callback
Orphan StringName: res://addons/gut/test.gd::944::double_script
Orphan StringName: _init_types_dictionary
Orphan StringName: test_assert_valid_paths
Orphan StringName: res://addons/gut/test.gd::870::set_double_strategy
Orphan StringName: minor
Orphan StringName: res://addons/gut/stub_params.gd::37::to_s
Orphan StringName: passing
Orphan StringName: index
Orphan StringName: res://addons/gut/stubber.gd::146::get_logger
Orphan StringName: ::8::TestListener.before_all
Orphan StringName: groups
Orphan StringName: replace_node
Orphan StringName: queue_free
Orphan StringName: line
Orphan StringName: FULL
Orphan StringName: indents
Orphan StringName: object_instance_id
Orphan StringName: res://addons/gut/thing_counter.gd::21::sum
Orphan StringName: music_file
Orphan StringName: val
Orphan StringName: sound_stopped
Orphan StringName: emitted
Orphan StringName: _summary
Orphan StringName: res://addons/gut/stubber.gd::152::to_s
Orphan StringName: GUT_METADATA
Orphan StringName: create_sound_instance
Orphan StringName: p10
Orphan StringName: path_or_node
Orphan StringName: res://addons/gut/test.gd::381::assert_does_not_have
Orphan StringName: element
Orphan StringName: get_listener_weight
Orphan StringName: path
Orphan StringName: res://addons/gut/test.gd::372::assert_has
Orphan StringName: listenerNum
Orphan StringName: test_assert_minimum_distance
Orphan StringName: get_event_volume
Orphan StringName: res://addons/gut/utils.gd::69::nvl
Orphan StringName: get_errors
Orphan StringName: ::57::TestEvent.test_assert_timeline_position
Orphan StringName: res://addons/gut/stubber.gd::143::clear
Orphan StringName: a
Orphan StringName: res://addons/gut/method_maker.gd::166::get_decleration_text
Orphan StringName: d
Orphan StringName: f
Orphan StringName: _double_scene
Orphan StringName: i
Orphan StringName: ::8::TestGlobal.before_all
Orphan StringName: _test_prefix
Orphan StringName: res://addons/gut/test.gd::660::assert_string_contains
Orphan StringName: override_path
Orphan StringName: p
Orphan StringName: default_args
Orphan StringName: ::43::TestEvent.test_assert_set_volume
Orphan StringName: s
Orphan StringName: t
Orphan StringName: timeline_marker
Orphan StringName: res://addons/gut/test.gd::867::get_double_strategy
Orphan StringName: test_assert_bank_bus_count
Orphan StringName: get_loaded_class
Orphan StringName: res://addons/gut/test.gd::835::yield_to
Orphan StringName: ::24::TestScript._init
Orphan StringName: res://addons/gut/utils.gd::82::is_freed
Orphan StringName: test_should_has_event
Orphan StringName: ::24::TestEventDescription.after_all
Orphan StringName: ::53::TestEvent.test_assert_paused
Orphan StringName: defaults
Orphan StringName: FMOD_STUDIO_LOAD_BANK_NORMAL
Orphan StringName: get_name
Orphan StringName: partial_double_inner
Orphan StringName: res://test/unit/test_event.gd::0::_init
Orphan StringName: res://addons/gut/doubler.gd::370::partial_double_scene
Orphan StringName: assert_eq
Orphan StringName: script_methods
Orphan StringName: res://addons/gut/signal_watcher.gd::105::watch_signals
Orphan StringName: res://test/unit/test_global.gd::0::_init
Orphan StringName: res://addons/gut/stubber.gd
Orphan StringName: list
Orphan StringName: res://test/unit/test_bus.gd
Orphan StringName: res://addons/gut/thing_counter.gd::15::get
Orphan StringName: VARIABLE_PROPERTY
Orphan StringName: nvl
Orphan StringName: _suppress_output
Orphan StringName: ::0::TestBank._init
Orphan StringName: NOT_SET
Orphan StringName: desiredValue
Orphan StringName: inst
Orphan StringName: get
Orphan StringName: res://addons/gut/test.gd::257::get_logger
Orphan StringName: _write_file
Orphan StringName: got_type
Orphan StringName: assert_gt
Orphan StringName: ::74::TestListener.assert_no_object_attached_to_listener
Orphan StringName: set_double_strategy
Orphan StringName: _double_count
Orphan StringName: built_ins
Orphan StringName: res://addons/gut/doubler.gd::408::delete_output_directory
Orphan StringName: res://addons/gut/doubler.gd
Orphan StringName: assert_true
Orphan StringName: add_local_method
Orphan StringName: begins_with
Orphan StringName: ::0::Test._init
Orphan StringName: res://addons/gut/spy.gd::50::get_call_parameters
Orphan StringName: get_basename
Orphan StringName: does_object_have_signal
Orphan StringName: res://addons/gut/test.gd::242::before_each
Orphan StringName: res://addons/gut/logger.gd::57::get_deprecated
Orphan StringName: assert_is
Orphan StringName: flags
Orphan StringName: test_assert_should_has_master_bus
Orphan StringName: _logs
Orphan StringName: ref_str
Orphan StringName: double_inner
Orphan StringName: test_assert_should_be_3d
Orphan StringName: called_with
Orphan StringName: res://test/unit/test_sound.gd
Orphan StringName: res://addons/gut/utils.gd::38::get_logger
Orphan StringName: check_event_path
Orphan StringName: ::51::TestScript.add_pass
Orphan StringName: expected
Orphan StringName: res://addons/gut/test.gd::59::_init_types_dictionary
Orphan StringName: get_call_list_as_string
Orphan StringName: result
Orphan StringName: res://addons/gut/stub_params.gd
Orphan StringName: res://addons/gut/logger.gd::97::get_gut
Orphan StringName: EDITOR_PROPERTY
Orphan StringName: set_sound_volume
Orphan StringName: is_double
Orphan StringName: did_add
Orphan StringName: method_meta
Orphan StringName: stop_event
Orphan StringName: ::28::TestScript.to_s
Orphan StringName: _get_func_text
Orphan StringName: assert_lt
Orphan StringName: get_full_name
Orphan StringName: set_listener_number
Orphan StringName: expect_low
Orphan StringName: value
Orphan StringName: unique_scripts
Orphan StringName: logger
Orphan StringName: alt_method
Orphan StringName: res://addons/gut/test.gd
Orphan StringName: res://addons/gut/method_maker.gd::185::get_spy_call_parameters_text
Orphan StringName: property
Orphan StringName: res://addons/gut/test.gd::572::assert_signal_emit_count
Orphan StringName: get_summary
Orphan StringName: test_assert_should_not_has_bus
Orphan StringName: tname
Orphan StringName: Stubber
Orphan StringName: property_usage
Orphan StringName: set_software_format
Orphan StringName: _Logger
Orphan StringName: assert_ne
Orphan StringName: scripts
Orphan StringName: cf
Orphan StringName: set_return
Orphan StringName: Doubler
Orphan StringName: open
Orphan StringName: with_this
Orphan StringName: resource_path
Orphan StringName: GDSCRIPT_CLASS
Orphan StringName: res://test/unit/test_listener.gd
Orphan StringName: set_method_strategy
Orphan StringName: _File
Orphan StringName: base_node
Orphan StringName: _get_temp_path
Orphan StringName: is_null_or_empty
Orphan StringName: res://addons/gut/stub_params.gd::27::when_passed
Orphan StringName: key
Orphan StringName: to_lower
Orphan StringName: get_test_prefix
Orphan StringName: things
Orphan StringName: desc_get_instance_list
Orphan StringName: set_event_paused
Orphan StringName: should
Orphan StringName: ::22::TestGlobal.after_all
Orphan StringName: test_assert_bank_event_count
Orphan StringName: dq
Orphan StringName: get_signal_list
Orphan StringName: ::29::TestCallbacks.test_assert_has_signals
Orphan StringName: StubParams
Orphan StringName: to_return
Orphan StringName: make_partial_double
Orphan StringName: GDNativeLibrary
Orphan StringName: res://addons/gut/doubler.gd::345::get_spy
Orphan StringName: res://addons/gut/test.gd::208::_fail_if_not_watching
Orphan StringName: res://addons/gut/method_maker.gd
Orphan StringName: desc_has_cue
Orphan StringName: set_gut
Orphan StringName: ::36::TestBus.test_assert_should_not_has_bus
Orphan StringName: res://addons/gut/doubler.gd::309::_double_script
Orphan StringName: res://addons/gut/thing_counter.gd::0::_init
Orphan StringName: got
Orphan StringName: res://addons/gut/test.gd::852::get_fail_count
Orphan StringName: output_dir
Orphan StringName: get_tree
Orphan StringName: find_last
Orphan StringName: LOG_LEVEL_FAIL_ONLY
Orphan StringName: assert_almost_eq
Orphan StringName: res://addons/gut/logger.gd::100::set_gut
Orphan StringName: res://addons/gut/spy.gd::69::call_count
Orphan StringName: res://test/unit/test_event.gd
Orphan StringName: res://addons/gut/test.gd::293::assert_almost_eq
Orphan StringName: res://addons/gut/signal_watcher.gd::129::print_object_signals
Orphan StringName: max_val
Orphan StringName: res://test/unit/test_zclose_fmod.gd
Orphan StringName: res://addons/gut/test.gd::799::assert_not_null
Orphan StringName: res://addons/gut/test.gd::315::assert_gt
Orphan StringName: check_bus_path
Orphan StringName: res://addons/gut/stubber.gd::15::_is_instance
Orphan StringName: test_assert_attach_object_to_listener
Orphan StringName: assert_null
Orphan StringName: test_assert_pause_unpause
Orphan StringName: res://test/unit/test_ainit_fmod.gd
Orphan StringName: res://addons/gut/test.gd::635::assert_is
Orphan StringName: obj
Orphan StringName: method_hash
Orphan StringName: res://addons/gut/summary.gd::106::get_totals
Orphan StringName: assert_contains_in_dict
Orphan StringName: ::0::TestGlobal._init
Orphan StringName: id
Orphan StringName: res://addons/gut/stub_params.gd::15::to_return
Orphan StringName: res://addons/gut/utils.gd::92::extract_property_from_array
Orphan StringName: test_assert_maximum_distance
Orphan StringName: type_flag
Orphan StringName: ::22::TestVCA.after_all
Orphan StringName: test_assert_should_set_listener_num
Orphan StringName: ::35::TestCallbacks.test_assert_set_callback
Orphan StringName: watch_signals
Orphan StringName: pause_before_teardown
Orphan StringName: res://addons/gut/test.gd::304::assert_almost_ne
Orphan StringName: res://addons/gut/doubler.gd::354::set_stubber
Orphan StringName: stop_all_bus_events
Orphan StringName: ::96::ObjectInfo.get_subpath
Orphan StringName: signal_name
Orphan StringName: test
Orphan StringName: res://addons/gut/logger.gd
Orphan StringName: ::32::TestEventDescription.test_assert_should_create_and_release
Orphan StringName: count
Orphan StringName: unmute_all_events
Orphan StringName: res://addons/gut/summary.gd::77::add_test
Orphan StringName: line_number
Orphan StringName: set_owner
Orphan StringName: set_event_parameter_by_name
Orphan StringName: file_path
Orphan StringName: import_tests
Orphan StringName: res://addons/gut/test.gd::700::assert_string_ends_with
Orphan StringName: get_super_call_text
Orphan StringName: time
Orphan StringName: res://addons/gut/test_collector.gd::201::set_test_class_prefix
Orphan StringName: gut
Orphan StringName: script_name
Orphan StringName: set_output_dir
Orphan StringName: res://addons/gut/test.gd::471::_find_object_property
Orphan StringName: ::37::TestVCA.test_assert_volume
Orphan StringName: sound_file
Orphan StringName: test_assert_mute_unmute
Orphan StringName: header
Orphan StringName: test_assert_bank_string_count
Orphan StringName: _fail_if_parameters_not_array
Orphan StringName: call_super
Orphan StringName: res://addons/gut/test.gd::895::_smart_double
Orphan StringName: test_assert_should_not_be_oneshot
Orphan StringName: get_class
Orphan StringName: res://test/unit/test_callbacks.gd
Orphan StringName: desc_is_one_shot
Orphan StringName: get_next
Orphan StringName: is_version_30
Orphan StringName: is_version_31
Orphan StringName: formatted
Orphan StringName: _watched_signals
Orphan StringName: ::68::TestListener.assert_listener_num
Orphan StringName: instance_list
Orphan StringName: ::30::TestSound.test_assert_set_pitch
Orphan StringName: _totals
Orphan StringName: get_current_script
Orphan StringName: append
Orphan StringName: memory
Orphan StringName: TestVCA
Orphan StringName: assert_signal_emitted
Orphan StringName: arg10
Orphan StringName: load_bank
Orphan StringName: arg11
Orphan StringName: res://addons/gut/test_collector.gd::189::set_logger
Orphan StringName: p1
Orphan StringName: p2
Orphan StringName: p3
Orphan StringName: p4
Orphan StringName: p5
Orphan StringName: p6
Orphan StringName: p7
Orphan StringName: ::116::DoubleInfo._init
Orphan StringName: p8
Orphan StringName: p9
Orphan StringName: res://addons/gut/doubler.gd::239::_get_stubber_metadata_text
Orphan StringName: assert_almost_ne
Orphan StringName: res://addons/gut/test.gd::391::assert_file_exists
Orphan StringName: oi
Orphan StringName: res://addons/gut/summary.gd::95::get_non_inner_class_script_count
Orphan StringName: unload_bank
Orphan StringName: text
Orphan StringName: after_all
Orphan StringName: get_bank_bus_count
Orphan StringName: MethodMaker
Orphan StringName: res://addons/gut/test_collector.gd::0::_init
Orphan StringName: res://addons/gut/test.gd::424::assert_file_not_empty
Orphan StringName: res://addons/gut/utils.gd::89::is_double
Orphan StringName: test_assert_should_have_memory_performance_data
Orphan StringName: res://test/unit/test_global.gd
Orphan StringName: _make_key_from_variant
Orphan StringName: get_max_count
Orphan StringName: DOUBLE_STRATEGY
Orphan StringName: pass_texts
Orphan StringName: res://addons/gut/doubler.gd::420::set_use_unique_names
Orphan StringName: res://addons/gut/test.gd::227::_fail_if_parameters_not_array
Orphan StringName: res://addons/gut/test.gd::199::_fail_if_does_not_have_signal
Orphan StringName: res://addons/gut/summary.gd
Orphan StringName: test_names
Orphan StringName: returns
Orphan StringName: ::8::TestBank.before_all
Orphan StringName: get_bank_string_count
Orphan StringName: get_infos
Orphan StringName: obj_info
Orphan StringName: get_parent
Orphan StringName: res://addons/gut/doubler.gd::250::_get_spy_text
Orphan StringName: get_version_info
Orphan StringName: TheScript
Orphan StringName: res://addons/gut/test.gd::152::_pass
Orphan StringName: get_extends_text
Orphan StringName: pop_back
Orphan StringName: ::40::TestBank.test_assert_bank_event_count
Orphan StringName: content
Orphan StringName: return_val
Orphan StringName: sp
Orphan StringName: res://addons/gut/logger.gd::103::clear
Orphan StringName: partial_double
Orphan StringName: res://addons/gut/test.gd::282::assert_ne
Orphan StringName: set
Orphan StringName: ::63::TestEventDescription.test_assert_should_not_have_cue
Orphan StringName: _test_order
Orphan StringName: metadata
Orphan StringName: object_info
Orphan StringName: ::42::ScriptMethods.add_built_in_method
Orphan StringName: instantiate
Orphan StringName: ts
Orphan StringName: import_from
Orphan StringName: res://addons/gut/test_collector.gd::207::has_script
Orphan StringName: pending_texts
Orphan StringName: keys
Orphan StringName: res://addons/gut/test.gd::861::get_assert_count
Orphan StringName: res://addons/gut/test.gd::346::assert_false
Orphan StringName: test_assert_volume
Orphan StringName: get_warnings
Orphan StringName: res://addons/gut/logger.gd::51::get_infos
Orphan StringName: get_path_to
Orphan StringName: ::27::TestScript.get_pass_count
Orphan StringName: test_assert_bus_stop_events
Orphan StringName: ARG_NOT_SET
Orphan StringName: is_sound_playing
Orphan StringName: ::46::TestGlobal.test_assert_should_have_file_performance_data
Orphan StringName: is_blacklisted
Orphan StringName: get_return
Orphan StringName: has_node
Orphan StringName: res://addons/gut/thing_counter.gd::6::add
Orphan StringName: add_call
Orphan StringName: res://addons/gut/method_maker.gd::101::_get_arg_text
Orphan StringName: res://addons/gut/test.gd::606::get_signal_parameters
Orphan StringName: res://addons/gut/test.gd::979::simulate
Orphan StringName: success
Orphan StringName: res://addons/gut/signal_watcher.gd::134::get_signal_parameters
Orphan StringName: res://addons/gut/logger.gd::81::info
Orphan StringName: res://addons/gut/test.gd::845::end_test
Orphan StringName: set_event_pitch
Orphan StringName: mute_all_events
Orphan StringName: wr
Orphan StringName: res://test/unit/test_sound.gd::0::_init
Orphan StringName: load_file_as_music
Orphan StringName: res://test/unit/test_zclose_fmod.gd::0::_init
Orphan StringName: ThingCounter
Orphan StringName: ::0::TestEvent._init
Orphan StringName: res://addons/gut/test.gd::873::pause_before_teardown
Orphan StringName: res://addons/gut/doubler.gd::337::get_output_dir
Orphan StringName: before_each
Orphan StringName: property_name
Orphan StringName: set_spy
Orphan StringName: has_inner_class
Orphan StringName: did
Orphan StringName: param_idx
Orphan StringName: has
Orphan StringName: dir
Orphan StringName: passed
Orphan StringName: res://addons/gut/doubler.gd::257::_get_func_text
Orphan StringName: desc_is_snapshot
Orphan StringName: get_event_timeline_position
Orphan StringName: get_totals
Orphan StringName: close
Orphan StringName: failing
Orphan StringName: is_freed
Orphan StringName: failed
Orphan StringName: BULLET
Orphan StringName: get_new
Orphan StringName: TestBus
Orphan StringName: _strategy
Orphan StringName: strategy
Orphan StringName: methods
Orphan StringName: test_assert_attached_to_node
Orphan StringName: ::66::TestScript._remap_path
Orphan StringName: res://addons/gut/signal_watcher.gd::110::watch_signal
Orphan StringName: Node
Orphan StringName: CPU
Orphan StringName: pause_all_events
Orphan StringName: set_logger
Orphan StringName: ::24::TestBus.after_all
Orphan StringName: _Mutex
Orphan StringName: _disable_strict_datatype_checks
Orphan StringName: Sprite
Orphan StringName: ::62::TestBus.test_assert_bus_stop_events
Orphan StringName: res://addons/gut/test.gd::849::get_summary
Orphan StringName: ::66::TestEventDescription.test_assert_minimum_distance
Orphan StringName: get_sound_pitch
Orphan StringName: not_expected
Orphan StringName: scripts_found
Orphan StringName: simulate
Orphan StringName: res://addons/gut/test.gd::246::after_all
Orphan StringName: has_printed_name
Orphan StringName: signals
Orphan StringName: ::0::TestCallbacks._init
Orphan StringName: length
Orphan StringName: _stub_to_call_super
Orphan StringName: write_file
Orphan StringName: FMOD_STUDIO_EVENT_CALLBACK_SOUND_STOPPED
Orphan StringName: ::83::TestEvent.test_assert_should_mute_unmute_all
Orphan StringName: res://addons/gut/doubler.gd::325::_double_scene
Orphan StringName: yield_to
Orphan StringName: res://addons/gut/test.gd::680::assert_string_starts_with
Orphan StringName: res://addons/gut/doubler.gd::233::_get_inst_id_ref_str
Orphan StringName: assert_has
Orphan StringName: _method_strategy
Orphan StringName: res://addons/gut/summary.gd::74::get_current_script
Orphan StringName: desired_value
Orphan StringName: spy
Orphan StringName: ::47::ScriptMethods.add_local_method
Orphan StringName: ::39::TestListener.test_assert_should_have_proper_weight
Orphan StringName: disconnect
Orphan StringName: res://addons/gut/spy.gd::84::get_call_list_as_string
Orphan StringName: res://test/unit/test_vca.gd
Orphan StringName: _get_fail_msg_including_emitted_signals
Orphan StringName: res://addons/gut/summary.gd::0::_init
Orphan StringName: double
Orphan StringName: _parse_inner_class_tests
Orphan StringName: file
Orphan StringName: TestEvent
Orphan StringName: res://addons/gut/summary.gd::68::add_script
Orphan StringName: get_performance_data
Orphan StringName: res://addons/gut/test.gd::260::set_logger
Orphan StringName: properties
Orphan StringName: ::72::ObjectInfo._init
Orphan StringName: replace_name
Orphan StringName: desc_get_sound_size
Orphan StringName: get_object_attached_to_instance
Orphan StringName: _fail_if_not_watching
Orphan StringName: save
Orphan StringName: res://addons/gut/test.gd::745::assert_not_called
Orphan StringName: _get_indented_line
Orphan StringName: FMOD_STUDIO_STOP_IMMEDIATE
Orphan StringName: res://addons/gut/doubler.gd::357::get_logger
Orphan StringName: get_bank_VCA_count
Orphan StringName: get_ref
Orphan StringName: res://addons/gut/test.gd::522::assert_signal_emitted
Orphan StringName: get_groups
Orphan StringName: find
Orphan StringName: asserts
Orphan StringName: assert_has_signal
Orphan StringName: to_do_nothing
Orphan StringName: print_object_signals
Orphan StringName: yield_for
Orphan StringName: assert_listener_weight
Orphan StringName: file_exists
Orphan StringName: res://addons/gut/signal_watcher.gd::147::is_watching
Orphan StringName: times
Orphan StringName: ::0::ScriptMethods._init
Orphan StringName: release_event
Orphan StringName: ftxt
Orphan StringName: ::22::TestListener.after_all
Orphan StringName: get_bus_paused
Orphan StringName: get_double_strategy
Orphan StringName: _parse_script
Orphan StringName: res://addons/gut/test.gd::133::_init
Orphan StringName: sum
Orphan StringName: _stubber
Orphan StringName: test_assert_event_reverb
Orphan StringName: res://addons/gut/test.gd::337::assert_true
Orphan StringName: _get_super_func_text
Orphan StringName: res://addons/gut/utils.gd::22::is_version_30
Orphan StringName: ::8::Test.to_s
Orphan StringName: res://addons/gut/summary.gd::71::get_scripts
Orphan StringName: res://addons/gut/test.gd::513::watch_signals
Orphan StringName: start_event
Orphan StringName: _get_methods
Orphan StringName: res://test/unit/test_ainit_fmod.gd::3::before_all
Orphan StringName: ::52::ScriptMethods.to_s
Orphan StringName: add_pass
Orphan StringName: res://addons/gut/doubler.gd::391::clear_output_directory
Orphan StringName: res://addons/gut/thing_counter.gd::27::to_s
Orphan StringName: res://addons/gut/method_maker.gd::92::_is_supported_default
Orphan StringName: test_assert_should_mute_unmute_all
Orphan StringName: variant
Orphan StringName: test_assert_has_signals
Orphan StringName: store_string
Orphan StringName: node_instance_id
Orphan StringName: res://addons/gut/test_collector.gd::195::set_test_prefix
Orphan StringName: bad_param_2
Orphan StringName: assert_file_not_empty
Orphan StringName: res://addons/gut/doubler.gd::274::_get_super_func_text
Orphan StringName: get_summary_text
Orphan StringName: get_strategy
Orphan StringName: _blacklist
Orphan StringName: get_bank_loading_state
Orphan StringName: load_file_as_sound
Orphan StringName: get_object_attached_to_listener
Orphan StringName: stub
Orphan StringName: utils
Orphan StringName: ::39::TestGlobal.test_assert_should_have_memory_performance_data
Orphan StringName: add_to_group
Orphan StringName: test_assert_timeline_position
Orphan StringName: set_stubber
Orphan StringName: ::108::ObjectInfo.get_method_strategy
Orphan StringName: expect_type
Orphan StringName: get_listener_number
Orphan StringName: _is_instance
Orphan StringName: empty
Orphan StringName: attach_instance_to_node
Orphan StringName: pad
Orphan StringName: YIELD
Orphan StringName: required_index
Orphan StringName: Test
Orphan StringName: name
Orphan StringName: unload_file
Orphan StringName: TestEventDescription
Orphan StringName: double_info
Orphan StringName: res://addons/gut/stubber.gd::43::_add_obj_method
Orphan StringName: res://addons/gut/test.gd::989::replace_node
Orphan StringName: res://addons/gut/test.gd::937::double_scene
Orphan StringName: assert_call_count
Orphan StringName: warn
Orphan StringName: get_bus_mute
Orphan StringName: res://addons/gut/doubler.gd::351::get_stubber
Orphan StringName: add_listener
Orphan StringName: get_method_strategy
Orphan StringName: res://addons/gut/test.gd::927::partial_double
Orphan StringName: res://addons/gut/test.gd::506::_can_make_signal_assertions
Orphan StringName: _supported_defaults
Orphan StringName: res://test/unit/test_ainit_fmod.gd::0::_init
Orphan StringName: res://addons/gut/test.gd::789::assert_null
Orphan StringName: set_strategy
Orphan StringName: res://test/unit/test_bank.gd::0::_init
Orphan StringName: res://addons/gut/spy.gd::95::set_logger
Orphan StringName: FMOD_STUDIO_LOADING_STATE_LOADED
Orphan StringName: set_event_volume
Orphan StringName: make_partial
Orphan StringName: get_non_inner_class_script_count
Orphan StringName: set_test_prefix
Orphan StringName: ::29::TestVCA.test_assert_valid_paths
Orphan StringName: res://addons/gut/stubber.gd::18::_make_key_from_metadata
Orphan StringName: WRITE
Orphan StringName: ::40::TestBus.test_assert_mute_unmute
Orphan StringName: test_assert_set_pitch
Orphan StringName: res://addons/gut/utils.gd::107::is_null_or_empty
Orphan StringName: res://addons/gut/utils.gd::101::write_file
Orphan StringName: ::33::ScriptMethods.is_blacklisted
Orphan StringName: call_count
Orphan StringName: add
Orphan StringName: test_assert_bank_vca_count
Orphan StringName: extract_property_from_array
Orphan StringName: get_VCA_volume
Orphan StringName: export_tests
Orphan StringName: ::54::TestGlobal.assert_contains_in_dict
Orphan StringName: res://addons/gut/doubler.gd::374::double_scene
Orphan StringName: extension
Orphan StringName: after_each
Orphan StringName: res://addons/gut/doubler.gd::137::_get_indented_line
Orphan StringName: assert_not_called
Orphan StringName: res://addons/gut/doubler.gd::317::_double_inner
Orphan StringName: _calls
Orphan StringName: set_bus_mute
Orphan StringName: usage
Orphan StringName: ::8::TestCallbacks.before_all
Orphan StringName: target_subpath
Orphan StringName: loc
Orphan StringName: list_dir_begin
Orphan StringName: get_logger
Orphan StringName: type
Orphan StringName: wanted
Orphan StringName: assert_string_starts_with
Orphan StringName: assert_exports
Orphan StringName: did_fail
Orphan StringName: get_stubber
Orphan StringName: ::70::TestEventDescription.test_assert_maximum_distance
Orphan StringName: get_property_list
Orphan StringName: set_value
Orphan StringName: get_doubler
Orphan StringName: to_call_super
Orphan StringName: res://addons/gut/signal_watcher.gd
Orphan StringName: get_spy_call_parameters_text
Orphan StringName: _gut
Orphan StringName: assert_file_exists
Orphan StringName: res://addons/gut/signal_watcher.gd::150::clear
Orphan StringName: res://addons/gut/doubler.gd::381::partial_double
Orphan StringName: ::58::TestScript.export_to
Orphan StringName: ::37::TestScript.get_new
Orphan StringName: ::48::TestBank.test_assert_bank_string_count
Orphan StringName: _method_maker
Orphan StringName: LOG_LEVEL_ALL_ASSERTS
Orphan StringName: ConfigFile
Orphan StringName: res://addons/gut/doubler.gd::153::_write_file
Orphan StringName: eof_reached
Orphan StringName: double_path
Orphan StringName: target
Orphan StringName: set_test_class_prefix
Orphan StringName: set_yield_signal_or_time
Orphan StringName: res://addons/gut/test.gd::487::assert_exports
Orphan StringName: res://addons/gut/doubler.gd::144::_stub_to_call_super
Orphan StringName: load
Orphan StringName: res://addons/gut/test.gd::878::get_summary_text
Orphan StringName: res://addons/gut/doubler.gd::302::_double
Orphan StringName: clear
Orphan StringName: create_event_instance
Orphan StringName: assert_signal_not_emitted
Orphan StringName: ext_loc
Orphan StringName: res://test/unit/test_bank.gd
Orphan StringName: test_assert_should_not_be_stream
Orphan StringName: file_perf_data
Orphan StringName: TestListener
Orphan StringName: res://addons/gut/test_collector.gd::146::_parse_inner_class_tests
Orphan StringName: parms_got
Orphan StringName: double_script
Orphan StringName: has_script
Orphan StringName: when_passed
Orphan StringName: remove_listener
Orphan StringName: _init
Orphan StringName: test_assert_should_pause_all
Orphan StringName: dest_path
Orphan StringName: all_supported
Orphan StringName: substr
Orphan StringName: connect
Orphan StringName: _tests
Orphan StringName: timeline_beat
Orphan StringName: ::23::TestSound.test_assert_set_volume
Orphan StringName: ::90::ObjectInfo.to_s
Orphan StringName: size
Orphan StringName: desc_release_all_instances
Orphan StringName: res://addons/gut/test.gd::951::double_inner
Orphan StringName: _test_class_prefix
Orphan StringName: res://addons/gut/utils.gd::44::split_string
Orphan StringName: ::46::TestScript.get_full_name
Orphan StringName: assert_no_object_attached_to_listener
Orphan StringName: partial_double_scene
Orphan StringName: replace
Orphan StringName: ::39::TestEvent.test_should_not_has_event
Orphan StringName: _get_spy_text
Orphan StringName: test_assert_should_have_file_performance_data
Orphan StringName: ::64::TestEvent.test_assert_event_reverb
Orphan StringName: array
Orphan StringName: res://addons/gut/test.gd::766::assert_call_count
Orphan StringName: delim
Orphan StringName: test_assert_should_not_have_cue
Orphan StringName: res://addons/gut/doubler.gd::367::set_strategy
Orphan StringName: node_instance
Orphan StringName: get_extension
Orphan StringName: res://addons/gut/method_maker.gd::174::get_super_call_text
Orphan StringName: add_test
Orphan StringName: res://addons/gut/method_maker.gd::199::set_logger
Orphan StringName: res://addons/gut/stubber.gd::131::should_call_super
Orphan StringName: ::93::ObjectInfo.get_path
Orphan StringName: _method_names
Orphan StringName: param_text
Orphan StringName: _double_inner
Orphan StringName: types
Orphan StringName: _pass
Orphan StringName: delete_output_directory
Orphan StringName: test_should_not_has_event
Orphan StringName: res://addons/gut/test.gd::355::assert_between
Orphan StringName: new_test
Orphan StringName: _path
Orphan StringName: assert_get_set_methods
Orphan StringName: res://addons/gut/utils.gd
Orphan StringName: ::48::TestEvent.test_assert_set_pitch
Orphan StringName: res://addons/gut/thing_counter.gd::34::get_max_count
Orphan StringName: res://addons/gut/summary.gd::80::add_pass
Orphan StringName: GDScript
Orphan StringName: music
Orphan StringName: @@14
Orphan StringName: @@15
Orphan StringName: @@16
Orphan StringName: sections
Orphan StringName: double_scene
Orphan StringName: @@18
Orphan StringName: @@19
Orphan StringName: res://addons/gut/test.gd::585::assert_has_signal
Orphan StringName: res://addons/gut/spy.gd::15::_get_params_as_string
Orphan StringName: _get_stubber_metadata_text
Orphan StringName: assert_between
Orphan StringName: play_sound
Orphan StringName: id2
Orphan StringName: id3
Orphan StringName: @@21
Orphan StringName: res://addons/gut/logger.gd::78::error
Orphan StringName: @@22
Orphan StringName: add_fail
Orphan StringName: @@24
Orphan StringName: @@25
Orphan StringName: @@27
Orphan StringName: res://addons/gut/test.gd::828::yield_for
Orphan StringName: res://addons/gut/spy.gd::0::_init
Orphan StringName: @@28
Orphan StringName: res://addons/gut/test.gd::966::stub
Orphan StringName: get_gut
Orphan StringName: test_assert_should_have_cpu_performance_data
Orphan StringName: res://addons/gut/thing_counter.gd::3::get_unique_count
Orphan StringName: sprite
Orphan StringName: PARAM_PREFIX
Orphan StringName: _remap_path
Orphan StringName: temp_path
Orphan StringName: @@30
Orphan StringName: log_type
Orphan StringName: get_fail_count
Orphan StringName: @@31
Orphan StringName: @@33
Orphan StringName: @@34
Orphan StringName: @@36
Orphan StringName: @@37
Orphan StringName: @@38
Orphan StringName: @@39
Orphan StringName: res://test/unit/test_bus.gd::0::_init
Orphan StringName: assert_has_method
Orphan StringName: res://addons/gut/test_collector.gd::186::get_logger
Orphan StringName: res://addons/gut/spy.gd::32::add_call
Orphan StringName: get_signal_parameters
Orphan StringName: has_subpath
Orphan StringName: detach_instance_from_node
Orphan StringName: res://addons/gut/utils.gd::86::is_not_freed
Orphan StringName: file_name
Orphan StringName: test_assert_invalid_path
Orphan StringName: stub_target
Orphan StringName: @@41
Orphan StringName: idx
Orphan StringName: match_case
Orphan StringName: double_strategy
Orphan StringName: object
Orphan StringName: res://addons/gut/logger.gd::60::get_count
Orphan StringName: res://addons/gut/test.gd::858::get_pending_count
Orphan StringName: test_class_prefix
Orphan StringName: ::17::TestSound.after_all
Orphan StringName: res://addons/gut/signal_watcher.gd::144::is_watching_object
Orphan StringName: get_subpath
Orphan StringName: set_bus_paused
Orphan StringName: FMOD_STUDIO_INIT_LIVEUPDATE
Orphan StringName: res://addons/gut/signal_watcher.gd::123::did_emit
Orphan StringName: res://addons/gut/signal_watcher.gd::80::_on_watched_signal
Orphan StringName: res://addons/gut/test.gd::438::assert_get_set_methods
Orphan StringName: thing
Orphan StringName: has_method
Orphan StringName: ScriptMethods
Orphan StringName: ::111::ObjectInfo.set_method_strategy
Orphan StringName: assert_string_ends_with
Orphan StringName: base_string
Orphan StringName: res://test/unit/test_desc_event.gd
Orphan StringName: res://addons/gut/signal_watcher.gd::0::_init
Orphan StringName: _double
Orphan StringName: _find_stub
Orphan StringName: res://addons/gut/logger.gd::84::debug
Orphan StringName: get_test_class_prefix
Orphan StringName: assert_signal_emitted_with_parameters
Orphan StringName: set_to
Orphan StringName: call_super_text
Orphan StringName: line_count
Orphan StringName: source
Orphan StringName: res://addons/gut/test.gd::326::assert_lt
Orphan StringName: assert_accessors
Orphan StringName: stubber
Orphan StringName: get_index
Orphan StringName: res://test/unit/test_vca.gd::0::_init
Orphan StringName: doubled
Orphan StringName: res://addons/gut/summary.gd::125::get_summary_text
Orphan StringName: res://addons/gut/stubber.gd::0::_init
Orphan StringName: TestSound
Orphan StringName: delta
Orphan StringName: res://addons/gut/stubber.gd::61::set_return
Orphan StringName: res://addons/gut/logger.gd::29::_log
Orphan StringName: should_call_super
Orphan StringName: _spy
Orphan StringName: add_script
Orphan StringName: _add_watched_signal
Orphan StringName: perf_data
Orphan StringName: get_line
Orphan StringName: res://addons/gut/stubber.gd::149::set_logger
Orphan StringName: set_sound_pitch
Orphan StringName: res://addons/gut/logger.gd::75::warn
Orphan StringName: make_dir_recursive
Orphan StringName: script
Orphan StringName: get_emit_count
Orphan StringName: ::47::TestBus.test_assert_pause_unpause
Orphan StringName: get_log_entries
Orphan StringName: desc_get_instance_count
Orphan StringName: ::0::TestSound._init
Orphan StringName: res://addons/gut/test.gd::553::assert_signal_emitted_with_parameters
Orphan StringName: method_strategy
Orphan StringName: assert_file_does_not_exist
Orphan StringName: assert_false
Orphan StringName: res://addons/gut/doubler.gd::178::_double_scene_and_script
Orphan StringName: release_sound
Orphan StringName: res://addons/gut/logger.gd::54::get_debugs
Orphan StringName: get_path
StringName: 1160 unclaimed string names at exit.
```

**Comment by 2shady4u** [comment]
Switching to latest GUT version seems to alleviate several leakage errors:
![image](https://user-images.githubusercontent.com/42484461/107572225-45a7a280-6bec-11eb-9336-f5d498c9bc9a.png)

I think these are solely fmod related?

**Comment by bitbrain** [comment]
Hopefully, a lot of these issues go away on the 4.x branch once the Godot 4 migration is complete (it contains an upgraded version of GUT too!)

**Comment by CedNaru** [comment]
Closing because old and GUT got updated plenty of time. Open another ticket if you encounter new issues with GUT.
